lib/node_modules/@stdlib/math/base/special/dirichlet-eta/lib/main.js

```diff
@@ -29,7 +29,7 @@ var LN2 = require( '@stdlib/constants/float64/ln-two' );
 // MAIN //
 
 /**
-* Evaluates the Dirichlet eta function.
+* Evaluates the Dirichlet eta function for a double-precision floating-point number `s`.
```

Comment:
Updated these descriptions here and everywhere else to include `double-precision`.